src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    static final int DATE_INDEX = 0;
+    static final int NAME_INDEX = 1;
+    static final int HOUR_INDEX = 2;
+    static final int RATE_INDEX = 3;
+    private SimpleDateFormat dateFormat = new SimpleDateFormat("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        dateFormat.setLenient(false);
+        try {
+            Date fromDate = dateFormat.parse(dateFrom);
+            Date toDate = dateFormat.parse(dateTo);
```

Comment:
checklist issue, Date is outdated since Java 8, use LocalDate instead
![image](https://github.com/mate-academy/jv-salary-info/assets/49767049/f8857e51-2034-491e-ae52-e8dcc46d67d6)